Resources/Locale/en-US/ghost/ghost-respawn.ftl

```diff
@@ -1,15 +1,15 @@
 ghost-respawn-time-left = Please wait {$time} {$time ->
     [one] minute
    *[other] minutes
-} before trying to return to the round.
+} before trying to respawn.
 
-ghost-respawn-max-players = Cannot return to the round right now. There should be fewer players than {$players}.
+ghost-respawn-max-players = Cannot respawn right now. There should be fewer than {$players} players.
 ghost-respawn-window-title = Respawn rules
 ghost-respawn-window-rules-footer = By respawning, you [color=#ff7700]agree[/color] [color=#ff0000]not to use any knowledge gained as your previous charactrer[/color]. Violation of this rule may constitute a server ban. Please, read the server rules for more details.
-ghost-respawn-same-character = You cannot enter the round as the same character. Please, change it in the character preferences.
+ghost-respawn-same-character = You cannot respawn as the same character. Please select a different one in character preferences.```
```

Comment:
On mobile so I can't suggest but the backticks from the suggestion block somehow got committed